core/foundation/res/ROOT/FoundationUtils.hxx

```diff
@@ -49,6 +49,17 @@ namespace FoundationUtils {
       return gPathSeparator;
    }
 
+   ///\returns the path separator for the PATH environment variable on the
+   /// platform.
+   inline const char& GetEnvPathSeparator() {
```

Comment:
How of curiosity, why not use a constexpr (and possibly a var rather than function)?

Reply:
Mostly for consistency with the function above. I guess both can become constexpr...